Android/app/src/main/java/fashiome/android/activities/MapFullScreenActivity.java

```diff
@@ -159,6 +169,44 @@ public void onMapReady(GoogleMap map) {
 
     }
 
+    public void loadProductsFromBackend(){
```

Comment:
I haven't changed anything in the models and parcels except for get photos method

Thanks,
Rakhee

On Mar 19, 2016, at 3:03 PM, Deepak Barge <notifications@github.com<mailto:notifications@github.com>> wrote:


In Android/app/src/main/java/fashiome/android/activities/MapFullScreenActivity.java<https://github.com/deborshisaha/ModaRenta/pull/57#discussion_r56755822>:

> @@ -159,6 +169,44 @@ public void onMapReady(GoogleMap map) {
>
>      }
>
> +    public void loadProductsFromBackend(){


Parceling does work across intents now for all models and all fields.

-
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/deborshisaha/ModaRenta/pull/57/files/b5a0e76ed0e25c34f4b9737bf76df95ed8676ffb#r56755822>